var/spack/repos/builtin/packages/ginkgo/package.py

```diff
@@ -46,12 +47,16 @@ class Ginkgo(CMakePackage, CudaPackage, ROCmPackage):
 
     depends_on("cmake@3.9:", type="build")
     depends_on("cuda@9:", when="+cuda")
+    depends_on("mpi", when="+mpi")
 
-    depends_on("rocthrust", type="build", when="+rocm")
+    depends_on("rocthrust", type="link", when="+rocm")
```

Comment:
rocthrust is a header-only library. What does it mean for it to be a link dependency?